python/lsst/afw/geom/ellipses/ellipse.cc

```diff
@@ -85,7 +86,13 @@ PYBIND11_PLUGIN(_ellipse) {
     clsEllipse.def("shift", &Ellipse::shift);
     clsEllipse.def("getParameterVector", &Ellipse::getParameterVector);
     clsEllipse.def("setParameterVector", &Ellipse::setParameterVector);
-//    clsEllipse.def("transform", (Transformer const (Ellipse::*)(AffineTransform const &) const) &Ellipse::transform);
+    clsEllipse.def(
+        "transform",
+        [](Ellipse const & self, AffineTransform const & t) -> Ellipse {
```

Comment:
`Transformer`, `Convolution`, and `GridTransform` are expression objects that don't translate to Python well (they require on assignment operators being able to invoke type conversion logic).  They shouldn't be seen in Python at all.